### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][NVDA][MAS4.2.1][IntelliJ][Wipro]: NVDA is announicng extra information after invoking "Add" button using shortcut key in Run on web app for Containers dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1498]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select created Project ->Azure->Run on web app for containers.
4. "Run on web app for Containers" dialog will open.
5. Tab to and select Use Existing web app /Create New Web App radio button.
6. Navigate to  "Add" button  then press "Alt+Insert" then observe the NVDA Behavior

**Expected Result:**
**NVDA Behavior:**
NVDA should announce
Add New Configuration  list
Run External tool  selected  1 of 7

**Actual Result:**
**NVDA Behavior:**
NVDA is announcing as 
Add (Alt+Insert)  button  checked  Add (Alt+Insert)
System  submenu  Space
sample - IntelliJ IDEA
Run on Web App for Containers  dialog  
Invalid Data: Please specify a valid Docker file.
Artifact
Azure Container Registry
Docker File
Server URL
Username
Password
Image and tag
Container Registry
Startup File
 /
Web App for Containers
Before launch: Build Artifacts, Activate tool window
Add New Configuration  list
Run External tool  selected  1 of 7
**Jaws Behavior**: Jaws is announcing as expected
Add (Alt+Insert) Button Pressed
To activate press spacebar.
Add New Configuration List view
Run External tool
1 of 7
To move to items use the Arrow keys.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![run on web app for containers-alt-insert](https://user-images.githubusercontent.com/22956130/36136773-183c4590-10b8-11e8-8616-e3579a71df2e.jpg)
![by using shortcut key for add nvda annoucning some extra information in run on web app for containers window](https://user-images.githubusercontent.com/22956130/36136789-263ebb32-10b8-11e8-984b-cdd431f64c38.jpg)

**Comment by Eskibear** [comment]
duplicate with #1500 
Content is provided by IntelliJ run/debug configuration, out of the toolkit's control.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749944
